libraries/AP_HAL_ChibiOS/hwdef/Aocoda-RC-H743Dual/README.md

```diff
@@ -90,12 +90,16 @@ The correct battery setting parameters are:
  - BATT_VOLT_PIN 10
  - BATT_CURR_PIN 11
  - BATT_VOLT_MULT 11
+ - BATT2_VOLT_PIN 18
```

Comment:
where is this pinned out? schematic shows no connection to PA4 pin on cpu

Reply:
J8 14P in sch, and it's VB2 in wiring diagram (middle right of the fig)

Reply:
![aocoda_h743dual_wiring_diagram](https://github.com/ArduPilot/ardupilot/assets/6076534/8d2b1676-0372-4d82-9004-1b183f0c1b52)
where on this diagram?perhaps you intend to make another pinout version in the future?

Reply:
okay I see it now....thanks